README.md

```diff
@@ -1562,12 +1562,12 @@ pets.include? 'cat'
 There are a few excellent resources on Rails style, that you should consider if
 you have time to spare:
 
-  * [The Rails 4 Way](http://www.amazon.com/The-Rails-Addison-Wesley-Professional-Ruby/dp/0321944275)
-  * [Ruby on Rails Guides](http://guides.rubyonrails.org/)
-  * [The RSpec Book](http://pragprog.com/book/achbd/the-rspec-book)
```

Comment:
This title is old and out of print, I changed it to "Effective Testing with RSpec 3".